tests/test_eval.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2022 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import sys
+from typing import Any
+
+import omegaconf as om
+import pytest
+
+# Add repo root to path so we can import scripts and test it
+repo_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+sys.path.append(repo_dir)
+
+from scripts.eval.eval import main  # noqa: E402
+
+
+@pytest.fixture(autouse=True)
+def set_correct_cwd():
```

Comment:
why do we need this?

Reply:
tasks.yaml contains a bunch of dataset paths that are relative to the scripts directory